utils/validation/user.ts

```diff
@@ -49,64 +49,85 @@ function validatePreferredLanguage(userLanguage) {
  * @param user - the user data to validate
  * @returns - true if the user data is valid, false otherwise
  */
-function userIsValid(user: UserInput): boolean {
+function getUserValidationErrors(user: UserInput): Array<string> {
     // validate base user fields
-    const firstNameIsAlpha = validator.isAlpha(user.firstName, undefined, " -");
-    const lastNameIsAlpha = validator.isAlpha(user.lastName, undefined, " -");
-    const roleIsValid = VALID_ROLES.has(user.role);
-
-    // validate the User fields
-    const userDataIsValid = firstNameIsAlpha && lastNameIsAlpha && roleIsValid;
+    const validationErrors = [];
+    if (!validator.isAlpha(user.firstName, undefined, " -")) {
```

Comment:
What is the 2nd and 3rd parameter of `isAlpha`?

Reply:
2nd parameter is the locale (defaults to `en-US`) and 3rd indicates which characters to allow - I actually used the 3rd param incorrectly here but I fixed it